lib/matplotlib/figure.py

```diff
@@ -1133,11 +1120,25 @@ def draw(self, renderer):
         dsu = [row for row in dsu if not row[1].get_animated()]
         dsu.sort(key=itemgetter(0))
 
-        mimage._draw_list_compositing_images(
-            renderer, self, dsu, self.suppressComposite)
+        try:
+            renderer.open_group('figure')
+            # prevent triggering call backs during the draw process
```

Comment:
Doesn't this comment refer to the `self._stale = True` line that was removed? I guess it possibly does nothing since it shouldn't have that underscore, but the comment should go too, if it's not going to be left in.